modules/ROOT/pages/dw-core-functions-now.adoc

```diff
@@ -45,7 +45,7 @@ output application/json
 This example shows uses of the `now()` function with valid
 selectors. It also shows how to get the epoch time with `now() as Number`.
 For additional examples, see
-https://docs.mulesoft.com/mule-runtime/4.3/dataweave-types#dw_type_dates[Date and Time (dw::Core Types)].
```

Comment:
TODO: This was changed to "latest" in the latest source.

* https://docs.mulesoft.com/mule-runtime/latest/dataweave-types#dw_type_dates[Date and Time (dw::Core Types)].